examples/agent/websocket/simple/Program.cs

```diff
@@ -169,7 +169,7 @@ await agentClient.Subscribe(new EventHandler<InjectionRefusedResponse>((sender,
                 {
                     Console.WriteLine($"----> {e} received");
                 }));
-                await agentClient.Subscribe(new EventHandler<InstructionsUpdatedResponse>((sender, e) =>
+                await agentClient.Subscribe(new EventHandler<PromptUpdatedResponse>((sender, e) =>
```

Comment:
@naomi-lgbt  The mic example isn't working for me, the mic won't pick up audio. I was going to see if I can fix it, since you have issues with mic detection on your linux machine